pyproject.toml

```diff
@@ -47,19 +47,19 @@ packages = [
 
 
 [tool.poetry.dependencies]
-python = "^3.7.0"
+python = "^3.6.0"
```

Comment:
This should reduce the minimum python version required to 3.6.

Even graphql-core-next will only require 3.6: https://github.com/graphql-python/graphql-core-next

So I think this relaxation is safe.

Reply:
Indeed, will do this, but currently problem is not in the requirements, but in the `dataclasses` module usage which appears only in 3.7.